src/backend/gpopt/gpdbwrappers.cpp

```diff
@@ -2590,7 +2592,8 @@ register_mdcache_invalidation_callbacks(void)
 								  (Datum) 0);
 }
 
-// Has there been any catalog changes since last call?
+// We reset the cache in case of a catalog change or if the transaction in which
```

Comment:
Commas ,in which ..,

Reply:
I'm wrong, comma is unneseccary here, your original sentence was totaly correct. Sorry